pxtsim/embed.ts

```diff
@@ -250,7 +250,6 @@ namespace pxsim {
         function receiveMessage(event: MessageEvent) {
             let origin = event.origin; // || (<any>event).originalEvent.origin;
             // TODO: test origins
-
```

Comment:
since this change does not help, best to remove it

Reply:
I've removed this change.